client/src/components/Opinions/ImageTwoOpinions.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
+
+export default function ImageTwo ({ poll }) {
```

Comment:
what's the diff between ImageTwo and ImageOne besides its main content? can we create just one component so that it's more reusable?

Reply:
good point. resolved